### DISCUSSION TITLE
Having maintainers per language

### DISCUSSION THREAD

**Discussion by waghanza** [the-benchmarker/web-frameworks#3723] [General]
Hi @the-benchmarker/web-frameworks,

In order to have a fair comparison, the various implementations in here **MUST** respect some rules :
+ disable logging
+ return empty strings on `GET:/` / `POST:/user`
+ returns the URL param on `GET:/user/$param"
+ be production-ready (not tuned for performances)

Unfortunately, I have not enough skills in all languages we are dealing with here.

My idea is to have some reviewers per languages, so as this project keeps its target.

Any volunteer for this ?

Regards,

PS : For me, it means that I will ask you for reviews and I won't merge until accepted by you ❤️ 

--- 
edit : I plan also to mention you on `README`

**Comment by dalisoft** [comment]
Hi @waghanza 
I can be volunteer and can do review, not always instant, but within 36-hours i can do review. 
i am happy to help

edit: only for javascript/typescript can

**Comment by System-Glitch** [comment]
Reviews would be for new libs/frameworks and updates?

Maybe I could review Go entries, if that doesn't take too much time. I just want to evaluate that because I don't want to commit to too many things at once.

**Reply by waghanza** [reply]
Indeed, on each update in one language, I'll ask for review. Sometimes, it could be just a version upgrade (`go`  **1.15** -> `go` **1.16**`) and then not big works, sometimes it will be to check the new frameworks in `go`

But, nothing is cast in stone, if you can not at some point, I'll try to find a solution, either by bypassing you or by waiting ❤️

**Reply by System-Glitch** [reply]
Nice, let's do that then!

**Comment by ahopkins** [comment]
:hand: I think this is a good idea. I am willing to do this for Python.

**Comment by pimbrouwers** [comment]
I agree with Adam and can help do this for F#.

**Reply by panesofglass** [reply]
I can help with C# and F#.

**Comment by whiplash** [comment]
✋ good idea. I am willing to do this for Scala.

**Comment by waghanza** [comment]
@0xTim would you mind volunteer for `swift` ?

**Reply by 0xTim** [reply]
Sure

**Comment by waghanza** [comment]
@leocavalcante @sy-records  would you mind volunteer for `php` ?

**Reply by sy-records** [reply]
ok

**Reply by leocavalcante** [reply]
sure

**Comment by pimbrouwers** [comment]
I'm sorry guys and gals, but I need to bow out of this endeavour. Something
has come up in my personal life that will sap all of my non-working life.

Apologies for the inconvenience. But I'd rather be upfront with you all,
then be a complete dead weight.

**Reply by waghanza** [reply]
sorry for you to hear that

no worries for us (at least for me) ... we all have our priorities

take your time, you will always be welcome here ❤️

**Comment by waghanza** [comment]
@lukaszlenart would you mind volunteer for `java` ?

**Comment by pimbrouwers** [comment]
Thanks Marwan.
